src/Init/GetElem.lean

```diff
@@ -82,9 +82,19 @@ class LawfulGetElem (cont : Type u) (idx : Type v) (elem : outParam (Type w))
    (dom : outParam (cont → idx → Prop)) [ge : GetElem cont idx elem dom] : Prop where
 
   getElem?_def (c : cont) (i : idx) [Decidable (dom c i)] :
-    c[i]? = if h : dom c i then some (c[i]'h) else none := by intros; eq_refl
+      c[i]? = if h : dom c i then some (c[i]'h) else none := by
+    intros
+    try simp only [getElem?, reduceDIte]
```

Comment:
This should be `reduceDite` I think.

Reply:
The capitalisation changed at some point, this is correct now.